source/documentation/upgrade_guide/topics/Installing_Virtual_Product_Hypervisors_on_local_RHEL.adoc

```diff
@@ -0,0 +1,88 @@
+sslclientcert =
+sslclientkey =
+----
++
+You need to populate the `sslclientcert` and `sslclientkey` fields with full pathnames to the correct files containing the appropriate certificate and key. The `/etc/pki/entitlement` directory contains one or more pair of certificate and key files, but only one pair contains the RHV-H entitlement you need.
+
+. To find the certificate file:
+
+.. List all files in the `/etc/pki/entitlement` directory:
++
+[options="nowrap" subs="normal"]
+----
+[root@rhel8testvm entitlement]# cd /etc/pki/entitlement/
+[root@rhel8testvm entitlement]# ls -al
+total 836
+drwxr-xr-x.  2 root root    202 May 28 15:18 .
+drwxr-xr-x. 15 root root    208 Apr 23  2020 ..
+-rw-r--r--.  1 root root   3243 May 28 15:18 4522783034260408538-key.pem
+-rw-r--r--.  1 root root 152622 May 28 15:18 4522783034260408538.pem
+-rw-r--r--.  1 root root   3243 May 28 15:18 5659494963772844103-key.pem
+-rw-r--r--.  1 root root 343394 May 28 15:19 5659494963772844103.pem
+-rw-r--r--.  1 root root   3243 May 23 13:19 645832581386032208-key.pem
+-rw-r--r--.  1 root root 343389 May 23 13:19 645832581386032208.pem
+[root@rhel8testvm entitlement]#
+----
+
+.. Use the `rct cat-cert` command on each certificate in order to find the one that contains the RHV-H entitlement:
++
+[options="nowrap" subs="normal"]
+----
+[root@rhel8testvm entitlement]# cd /etc/pki/entitlement/
+[root@rhel8testvm entitlement]# rct cat-cert 5659494963772844103.pem | grep rhvh/4/ | grep URL
+        URL: /content/beta/rhel/server/7/$basearch/rhvh/4/os
+        URL: /content/dist/rhel/server/7/7Server/$basearch/rhvh/4/os
+        URL: /content/beta/layered/rhel8/x86_64/rhvh/4/os
+        URL: /content/dist/layered/rhel8/x86_64/rhvh/4/os
+----
+
+. Identify the correct certificate and fill in the `sslclientcert` and `sslclientkey` values in the previously mentioned `.repo` file:
++
+[options="nowrap" subs="normal"]
+----
+.
+.
+.
+sslclientcert = /etc/pki/entitlement/5659494963772844103.pem
+sslclientkey = /etc/pki/entitlement/5659494963772844103-key.pem
+----
+
+. Run the `reposync` command in an appropriate directory:
++
+[options="nowrap" subs="normal"]
+----
+[root@rhel8testvm rhvh-reposync]# pwd
+/home/test/rhvh-reposync
+[root@rhel8testvm rhvh-reposync]# reposync --repo rhvh-4-for-rhel-8-x86_64-rpms
+Updating Subscription Management repositories.
+Red Hat Virtualization Host for RHEL 8 x86_64 (RPMs)                             11 kB/s | 4.0 kB     00:00
+Red Hat Virtualization Host for RHEL 8 x86_64 (RPMs)                            272 kB/s | 291 kB     00:01
+.
+.
+.
+(193/194): redhat-virtualization-host-image-update-4.4.5-20210330.0.el8_3.noarc 5.4 MB/s | 822 MB     02:30
+(194/194): rhvm-appliance-4.4-20210310.0.el8ev.x86_64.rpm                       5.6 MB/s | 1.5 GB     04:34
+[root@rhel8testvm rhvh-reposync]#
+----
+
+. Check the certificate and key file pairs with each `reposync` command because the Subscription Manager subsystem periodically regenerates them.
```

Comment:
@stoobie -- I added "command" to line 88. Do you agree or is there better phrasing/

Reply:
@RichardHoch 
I would say "...each time you run the `reposync` command."